pkgs/applications/misc/1password/default.nix

```diff
@@ -1,50 +1,57 @@
-{ stdenv, fetchzip, autoPatchelfHook, fetchurl, xar, cpio }:
+{ stdenv, fetchzip, fetchurl, autoPatchelfHook, xar, cpio, installShellFiles }:
 
 stdenv.mkDerivation rec {
   pname = "1password";
-  version = "1.1.1";
+  version = "1.2.0";
   src =
     if stdenv.isLinux then fetchzip {
       url = {
         "i686-linux" = "https://cache.agilebits.com/dist/1P/op/pkg/v${version}/op_linux_386_v${version}.zip";
         "x86_64-linux" = "https://cache.agilebits.com/dist/1P/op/pkg/v${version}/op_linux_amd64_v${version}.zip";
       }.${stdenv.hostPlatform.system};
       sha256 = {
-        "i686-linux" = "1andl3ripkcg4jhwdkd4b39c9aaxqpx9wzq21pysn6rlyy4hfcb0";
-        "x86_64-linux" = "0qj5v8psqyp0sra0pvzkwjpm28kx3bgg36y37wklb6zl2ngpxm5g";
+        "i686-linux" = "1fpmglqj7qqaa7k79ngrf16qq6rv4m8b57hzq87fyj22myaxydjy";
+        "x86_64-linux" = "0c0zj3bv0rgdqaanm6gk7fsy5ckj5h8zc5jckrv1nsfqr7lpbydd";
       }.${stdenv.hostPlatform.system};
       stripRoot = false;
     } else fetchurl {
       url = "https://cache.agilebits.com/dist/1P/op/pkg/v${version}/op_darwin_amd64_v${version}.pkg";
-      sha256 = "16inwxkrky4xwlr7vara1l8kapdgjg3kfq1l94i5855782hn4ppm";
+      sha256 = "0sc5fi55gcbjmj5cp9clilkdz7c6nzqyyh1fficfafh8il4qvki6";
     };
 
+  nativeBuildInputs = [ installShellFiles ]
+  ++ stdenv.lib.optionals stdenv.isLinux [ autoPatchelfHook ];
```

Comment:
I think it's more conventional to indent the `++` here with two spaces. You could check with `nixpkgs-fmt`.